src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -453,17 +454,42 @@ private boolean stringToBoolean(String value) {
 
     @JsonIgnore
     public SscsDocument getLatestDocumentForDocumentType(DocumentType documentType) {
-        if (getSscsDocument() != null && getSscsDocument().size() > 0) {
 
-            Stream<SscsDocument> filteredDocs = getSscsDocument().stream()
+        if (getSscsDocument() != null && getSscsDocument().size() > 0) {
+            Stream<SscsDocument> filteredStream = getSscsDocument().stream()
                 .filter(f -> documentType.getValue().equals(f.getValue().getDocumentType()));
 
-            List<SscsDocument> docs = filteredDocs.sorted().collect(Collectors.toList());
+            List<SscsDocument> filteredList = filteredStream.collect(Collectors.toList());
 
-            if (docs.size() > 0) {
-                return docs.get(0);
+            Collections.sort(filteredList, (one, two) -> {
```

Comment:
There is another way to sort by using CompareToBuilder() e.g new CompareToBuilder()
                .append(this.value.getHearingDateTime(), o.getValue().getHearingDateTime())
                .toComparison();